ci/Dockerfile-envoy-google-vrp

```diff
@@ -1,4 +1,41 @@
-FROM envoyproxy/envoy:local
+ARG BUILD_FROM=ubuntu:18.04
```

Comment:
Why do you need this change? you don't really need this?

Reply:
Hi, Lizan
The envoy-google-vrp is based on envoyproxy/envoy:local image. But when building envoy-google-vrp by buildx, I found that the buildx can not find the envoyproxy/envoy:local image which was just built by buildx. The envoyproxy/envoy:local image does not exist in docker images list. 
If we export envoyproxy/envoy:local by command "docker buildx build -o type=docker ", buildx still can not access that image. Although "docker build" could access envoyproxy/envoy:local image.
So, I think, if we replace based image by source dockerfile, the build script may be simpler than before. At same time, for envoy-google-vrp image, it could be more dependent. And it also does not extend the building time since it will use the image cache which was built before.

Reply:
Will defer this part to @htuch

Reply:
Seem plausible. FWIW, I don't think we need VRP for ARM, but if you'd like to cleanup the build flow while retaining the same semantics (i.e. have the equivalent of `envoyproxy/envoy` latest state based on the current tree as the base) then that's great.

Reply:
@lizan Have updated. Please check it.